lib/reunion.rb

```diff
@@ -0,0 +1,37 @@
+class Reunion
+  attr_reader :name, :activities
+ def initialize(name)
+   @name = name
+   @activities = []
+ end
+
+ def add_activity(activity)
+   @activities << activity
+ end
+
+ def total_cost
+   @activities.sum do |activity|
+     activity.total_cost
+   end
+ end
+
+ def breakout
+   almost_there = @activities.map do |activity|
+     activity.owed.transform_values do |owe|
```

Comment:
This transform_values isn't doing anything since it's transforming the value to itself. Instead, you could just call activity.owed